### PR TITLE
Adding changes for 10.80

### PR DESCRIPTION
**Pull request by mfreeman451**
Fixed a bug that was preventing this from working with OO 10.80, added ability to get execution summary from OO API

### CODE REVIEW

File: lib/Executions.js [hunk 1]
```diff
@@ -22,6 +22,20 @@ class Executions {
 
     return result
   };
+
+	async executionSummary (flowid, cb) {
```
Comment (lirantal) [inline lib/Executions.js]: can you please update the indentations to match the file standards? (i.e: 2 spaces)

**Comment by lirantal** [comment]
@mfreeman451 Thank you for your first contribution!
Could you please fix the lint issues that the build fails on so I can merge it? https://travis-ci.org/lirantal/operations-orchestration-api/builds/361491334?utm_source=github_status&utm_medium=notification

**Review by lirantal — CHANGES_REQUESTED** [review]

**Review by mfreeman451 — COMMENTED** [review]
Updated, do I just generate a new pull request?

**Comment by mfreeman451** [comment]
Fixes in place, how to proceed?

**Comment by lirantal** [comment]
@mfreeman451 can you take a look at why the tests fail now?
https://travis-ci.org/lirantal/operations-orchestration-api/builds/364170715?utm_source=github_status&utm_medium=notification

**Comment by mfreeman451** [comment]
Updated test.js

**Comment by lirantal** [comment]
@mfreeman451 are you sure you pushed the relevant update?
the test still fails - check it out here: https://travis-ci.org/lirantal/operations-orchestration-api/builds/364330226?utm_source=github_status&utm_medium=notification

**Comment by lirantal** [comment]
Closing due to stale PR but if you want to get back at it @mfreeman451 please open a new one and I'll be happy to merge quickly.

**Pull request closed without merging**